index.html

```diff
@@ -1,13 +1,246 @@
+          </ul>
+          <a class=buttonlink href="https://codepen.io/PowerLevel9000/pen/poKxGGW" target="_blank" rel="noopener noreferrer"><button>See Project</button></a>
+        </div>
+      </div>
+
+
+      <div class="project one">
+        <img src="projects-img/cyber-security.png" alt="introduction-form">
+        <div>
+          <h2 class="project-title">Adarsh&apos;s introduction-form</h2>
+          <ul class="project-maker">
+            <li class="project-owner">Adarsh</li>
+            <li>future fullstack</li>
+            <li>2022</li>
+          </ul>
+          <p class="description info">
+            This is my cyber-security Document, I have made this with the tempplate 
+            of freecode camp documentation project 
+          </p>
+          <ul class="tech">
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <a class=buttonlink href="https://codepen.io/PowerLevel9000/pen/poKxGGW" target="_blank" rel="noopener noreferrer"><button>See Project</button></a>
+        </div>
+      </div>
+
+      <div class="project one">
+        <img src="projects-img/tribute-project.png" alt="tribute-project">
+        <div>
+          <h2 class="project-title">Bhardvaj's tribute</h2>
+          <ul class="project-maker">
+            <li class="project-owner">Adarsh</li>
+            <li>future fullstack</li>
+            <li>2022</li>
+          </ul>
+          <p class="description info">
+            This is a trubute page for indian bhardvaj rish who is the father ofguru of 
+            of guru of pandavas and made many resarches
+          </p>
+          <ul class="tech">
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <a class=buttonlink href="https://codepen.io/PowerLevel9000/pen/ExReWZm" target="_blank" rel="noopener noreferrer"><button>See Project</button></a>
+        </div>
+      </div>
+
+      <div class="project one">
+        <img src="projects-img/css-animation-1.png" alt="waveing panguin">
+        <div>
+          <h2 class="project-title">Waveing panguin</h2>
+          <ul class="project-maker">
+            <li class="project-owner">Adarsh</li>
+            <li>future fullstack</li>
+            <li>2022</li>
+          </ul>
+          <p class="description info">
+            I made this panguin css-animation when i was learning css month ago with free code camp
+            and i love this project do checkout
+          </p>
+          <ul class="tech">
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <a class=buttonlink href="https://codepen.io/PowerLevel9000/pen/mdKGWej" target="_blank" rel="noopener noreferrer"><button>See Project</button></a>
+        </div>
+      </div>
+
+    </section>
+
+    <section class="aboutme">
+      <h2>About <br> Myself</h2>
+      <p class="description info2" style="padding-left: 20px;">
+        Hello I&apos;m a software developer! I can help you 
```

Comment:
- [ ] Turn to the about section to receive some feedback 😄 , 
<img width="632" alt="s4" src="https://user-images.githubusercontent.com/77843050/205439686-36ff8806-7f1e-4568-9b07-c4e1b1d54784.png">

- [ ] There we go, 1. The space that separates the work section from the about has to be a bit adjusted, kindly fix that.
- [ ] 2. the top of the `about my self` title is mismatched with the one on the original design, please update that too.
- [ ] 3. Your paragraph text does not have the same color and size, isn't it? Yes. so, please fix it.
- [ ] 4. Your `let's connect` text is a bit big than the one on the original design, kindly make a few touches to it.
- [ ] 5. To finish, we have some spaces on the left and the right of the about section which is not what we have and see on the original design, please fix that up.